main.java

```diff
@@ -11,33 +13,67 @@ public static void main(String[] args) {
 
         System.out.println("Choose your magical creature to use for battle!");
         System.out.println("-".repeat(93));
-        //list of creatures
-        System.out.println("Troll");
-        System.out.println("Dragon");
-        System.out.println("Hippogriff");
-        System.out.println("Unicorn");
-        System.out.println("Cerberus");
-        System.out.println("Mandrake");
-
-//user input
+         
+     //creating the creatures 
+        var myDragon = new Dragon();
+        var myHippogriff = new Hippogriff();
+        var myMandrake = new Mandrake();
+        var myThreeHeadedDog = new ThreeHeadedDog();
+        var myTroll = new Troll();
+        var myUnicorn = new Unicorn();
+
+
     System.out.println();
-    Scanner myObj = new Scanner(System.in);
+
+    //added scanner to scane user input 
+    Scanner input = new Scanner(System.in);
+
+    //make array of creatures to number
+        ArrayList<Creature> availableCreatures = new ArrayList<Creature>();  
```

Comment:
shorthand... to reduce redudancy
```suggestion
        var availableCreatures = new ArrayList<Creature>();  
```